packages/pagedown/install

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+set -x
+set -e
+
+# the functional test has failed without Pandoc
```

Comment:
pandoc is available on shinyapps.io, I will add it to the Dockerfile which is the better place for it to be.

Reply:
#226

Reply:
https://github.com/rstudio/shinyapps-package-dependencies/pull/220/commits/ae3445efb39cd1e6af36b270945633e16e93bcbb